bin/xml/impl/glassfish/common.xml

```diff
@@ -1434,19 +1434,19 @@
     -->
     <target name="create.jacc.vm.options" depends="configPlatform">
          <antcall target="create-jvm-options" >
-          <param name="jvm.options" value=" -Djavax.security.jacc.policy.provider=com.sun.ts.tests.jacc.provider.TSPolicy"/>
+          <param name="jvm.options" value=" -Djakarta.security.jacc.policy.provider=com.sun.ts.tests.jacc.provider.TSPolicy"/>
```

Comment:
Just for the record, these will create new failures during the setup but inevitable. These would be fixed by using the Glassfish  bundles with jakarta changes.

Reply:
thanks @alwin-joseph!  I created [issue 239 for ](https://github.com/eclipse-ee4j/jakartaee-tck/issues/239) tracking this.